rust/chains/hyperlane-cosmos/src/payloads/general.rs

```diff
@@ -0,0 +1,24 @@
+use serde::{Deserialize, Serialize};
+
+#[derive(Serialize, Deserialize, Debug)]
+pub struct EmptyStruct {}
```

Comment:
I assume serde treats empty struct and unit structs identically, could we use a unit struct here for conciseness?